src/main/java/dev/galacticraft/mod/content/block/machine/SimpleMultiBlockMachineBlock.java

```diff
@@ -24,15 +24,20 @@
 
 import dev.galacticraft.mod.api.block.MultiBlockMachineBlock;
 import dev.galacticraft.mod.api.block.MultiBlockPart;
+import dev.galacticraft.mod.content.GCBlocks;
+import dev.galacticraft.mod.content.block.special.SolarPanelPartBlock;
 import net.minecraft.core.BlockPos;
 import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Explosion;
 import net.minecraft.world.level.Level;
 import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.block.entity.BlockEntity;
 import net.minecraft.world.level.block.state.BlockState;
 import org.jetbrains.annotations.Unmodifiable;
 
 import java.util.List;
+import java.util.function.BiConsumer;
 
 public class SimpleMultiBlockMachineBlock extends MultiBlockMachineBlock {
```

Comment:
It would be better to implement this on `MultiBlockMachineBlock` so that all multiblocks can benefit from this change.